### ISSUE TITLE
Build fails

### ISSUE THREAD

**Issue by bpross-52n** [opengeospatial/teamengine#510]
**Describe the bug**
TECoreTest.testNestedFailure() falis during build

**To Reproduce**
1. Build master branch
2. See error

**Expected behavior**
Build is successful

**Additional context**
Seems to be caused by https://github.com/opengeospatial/teamengine/pull/474

**Comment by hwbllmnn** [comment]
You're right, my 'fix' apparently only fixed our failing use case. In #518 I added a similar test case you linked to and fixed the issue hopefully once and for all.
